aws/rust-runtime/aws-runtime/Cargo.toml

```diff
@@ -33,16 +33,19 @@ tracing = "0.1"
 uuid = { version = "1" }
 
 [dev-dependencies]
+arbitrary = "1.3"
```

Comment:
We should be able to remove `arbitrary` from `[dev-dependencies]` in `aws-config`'s `Cargo.toml`